dotCMS/src/main/java/com/dotcms/util/SecurityUtils.java

```diff
@@ -283,5 +289,18 @@ protected List<String> loadIgnoreHosts() {
     }
     return IGNORE_REFERER_FOR_HOSTS;
   }
-  
+
+
+  /**
+   * Validate if the fileName is secure and valid
+   * @param fileName  {@link String}
+   */
+  public void validateFileName (final String fileName) {
```

Comment:
Maybe we can change the method's name, as it is not only validating the file's name, but also its path